### PR TITLE
adds multiple trips query

### PR DESCRIPTION
**Pull request by ErinLMoore**
Asana task: https://app.asana.com/0/1167196461144361/1204898712337424/f

Screenshot 2023-07-18 at 2 52 46 PM

### CODE REVIEW

File: apps/alert_processor/lib/model/query.ex [hunk 1]
```diff
@@ -246,6 +246,28 @@ defmodule AlertProcessor.Model.Query do
...
+            LEFT JOIN subscriptions s on s.trip_id = s.id
+             WHERE t.id = ANY(
+              SELECT DISTINCT s.trip_id FROM subscriptions s
+              WHERE s.paused = false
+            )
+            GROUP BY u.id
+            HAVING COUNT(t.id) = i
+          )x
+          HAVING COUNT(x) > 0
+        ) t;
```
Comment (arkadyan) [inline apps/alert_processor/lib/model/query.ex:269]: nitpick: Same comment as above with duplicated names.

**Review by arkadyan — COMMENTED** [review]
I think this looks good, and once I wrap my head around it not too crazy. A couple of tiny notes, but the one big thing is the question about the `id`—am I getting confused about that?

File: apps/alert_processor/lib/model/query.ex [hunk 2]
```diff
@@ -246,6 +246,28 @@ defmodule AlertProcessor.Model.Query do
...
+        label: "Count of Active Users By Number of Active Trips",
+        query: """
+        SELECT t.*
+        FROM  generate_series(0,50) i
+        CROSS JOIN LATERAL (
+          SELECT i AS active_trips, COUNT(x.*) AS active_users FROM
+          (
+            SELECT u.id FROM users u
+            JOIN trips t ON t.user_id = u.id
+            LEFT JOIN subscriptions s on s.trip_id = s.id
```
[the remark this answers is not in the document]
Comment (arkadyan) [reply apps/alert_processor/lib/model/query.ex:266]: Yeah, that's why I was wondering if I was missing something…

[the remark this answers is not in the document]
Comment (ErinLMoore) [reply apps/alert_processor/lib/model/query.ex:266]: update: the original way definitely gives back the correct data, but i feel like i have to figure out why now.

**Review by arkadyan — APPROVED** [review]
👍  it might need a rebase, but looks good to me

**Pull request merged**